src/main/flight/imu.c

```diff
@@ -87,14 +86,18 @@ static float throttleAngleScale;
 static float fc_acc;
 static float smallAngleCosZ = 0;
 
-static float magneticDeclination = 0.0f;       // calculated at startup from config
```

Comment:
That particular variable is not used, however magnetic declination also exists in `mag_t` (`compass.h`).
However, that instance is only ever set at the moment.